test-suite/report.sh

```diff
@@ -21,7 +21,7 @@ cp summary.log "$SAVEDIR"/
 rm "$FAILED"
 
 # print info
-if [ -n "$APPVEYOR" ] || [ -n "$PRINT_LOGS" ]; then
+if [ -n "$CI" ] || [ -n "$PRINT_LOGS" ]; then
```

Comment:
I still wonder if we shouldn't do this in general? In `test-suite/dune` we set it unconditionally as indeed I _always_ want to have PRINT_LOGS=1 when developing locally too.

Reply:
It can get pretty verbose, and locally the logs are easily gotten (or just run `make report PRINT_LOGS=1`)